src/code/game.c

```diff
@@ -77,14 +77,16 @@ void func_800C4344(GameState* gameState) {
     if (D_80121212 != 0) {
         func_801C7E78();
     }
-#elif OOT_DEBUG
+#elif DEBUG_FEATURES
```

Comment:
does it make sense to enable this for N64 too by changing `#elif` to `#endif + #if`? (Maybe also moving the N64 block down so the scoping syntax works out)